app-arch/innoextract/innoextract-1.5.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils toolchain-funcs cmake-utils
+
+DESCRIPTION="A tool to unpack installers created by Inno Setup"
+HOMEPAGE="http://constexpr.org/innoextract/"
+SRC_URI="http://constexpr.org/innoextract/files/${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="c++0x debug doc +iconv +lzma static"
+
+RDEPEND="
+	!static? (
+		dev-libs/boost:=
+		iconv? ( virtual/libiconv )
+		lzma? ( app-arch/xz-utils )
+	)"
+DEPEND="${RDEPEND}
+	doc? ( >=app-doc/doxygen-1.8.3.1 )
+	static? (
+		app-arch/bzip2[static-libs]
+		dev-libs/boost[static-libs]
+		sys-libs/zlib[static-libs]
+		iconv? ( virtual/libiconv )
```

Comment:
`virtual/libiconv` is broken, since it doesn't allow to propagate `static-libs` USE flag to `dev-libs/libiconv`. One way to fix that would probably be to do
```
    iconv? (
        !elibc_glibc? ( !elibc_uclibc? ( !elibc_musl? ( !elibc_mintlib? (
            || ( >=dev-libs/libiconv-1.14-r1[static-libs] >=sys-freebsd/freebsd-lib-10.0 )
        ) ) ) )
    )
```
But then, I'm not even sure if `sys-freebsd/freebsd-lib` supports static libs.

@mgorny do you have an idea?